### ISSUE TITLE
[sprintf] unexpected placeholder error on Progress report page

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4951]
## Summary

Getting this (again?) on latest .deb installer. Happens in all locales.

## Branch or installer method

Beta1 .deb with latest contentpacks on Ubuntu 15.10

## Traceback or relevant snippet from server.log

I can't seem to find any red flags in the server.log:
```
[INFO] [2016-03-02 22:15:29,646] django.request: HTTP Request /api/attemptlog/ - Response: 201
[INFO] [2016-03-02 22:15:29,672] django.request: HTTP Request /api/attemptlog/bc995a42419148f8b222005d354670b2/ - Response: 200
[INFO] [2016-03-02 22:15:29,748] django.request: HTTP Request /api/exerciselog/fc5f933a709c5273b15d3f04f2a75f9c/ - Response: 204
[INFO] [2016-03-02 22:15:48,592] django.request: HTTP Request / - Response: 200
[INFO] [2016-03-02 22:15:48,665] cherrypy.access.139640908275472: 127.0.0.1 - - [02/Mar/2016:22:15:48] "GET /static/fonts/style.css HTTP/1.1" 304 - "http://127.0.0.1:8008/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/48.0.2564.116 Chrome/48.0.2564.116 Safari/537.36"
```

## How to reproduce

1. Log in as Learner. 
1. Do some exercises & videos.
1. Check `My progress` page.

## Screenshots
![sprintf](https://cloud.githubusercontent.com/assets/1457929/13476159/c78308fa-e0c6-11e5-9a09-db2b773362f3.png)

**Comment by radinamatic** [comment]
Pinging @aronasorman @rtibbles, in case you have the `release blocker` filter permanently on... :wink:

**Comment by radinamatic** [comment]
Confirming this on latest OSX installer (build 74).
This should be a `release blocker` @aronasorman @rtibbles.

**Comment by rtibbles** [comment]
Replicated.

**Comment by radinamatic** [comment]
Fix confirmed on WinXP (build 167).

**Comment by rtibbles** [comment]
Odd. PR that has the fix isn't merged?

**Comment by radinamatic** [comment]
Original issue was reported on the .deb installer, so maybe wait until it's merged there?

**Comment by rtibbles** [comment]
It's not merged anywhere, is what I am saying, so I am wondering how this is fixed in any build.

**Comment by radinamatic** [comment]
Gremlin(s)?

**Comment by radinamatic** [comment]
Must be... No errors on WinXP (build 167).

![ie8 - winxp 0 16 build 167 running - oracle vm virtualbox_261](https://cloud.githubusercontent.com/assets/1457929/13570576/5090a00c-e46f-11e5-8da0-c3f7b82b0cfc.png)

![ie8 - winxp 0 16 build 167 running - oracle vm virtualbox_262](https://cloud.githubusercontent.com/assets/1457929/13570581/5a01af82-e46f-11e5-8c0b-2c57f14086d0.png)


The only problem I'm seeing is that exercise titles do not show in Spanish, with the contentpack installed.

**Comment by rtibbles** [comment]
Fixed in #4963
